halo/attest/keeper/keeper_internal_test.go

```diff
@@ -0,0 +1,317 @@
+package keeper
```

Comment:
Why call this test file `keeper_internal_test.go` and not `keeper_test.go`? 
I have the impression the rest of the repository follows the latter pattern